app/controllers/boards_controller.rb

```diff
@@ -22,13 +22,25 @@ def edit
 
   def update
     @board = Board.find(params[:id])
-    if @board.update
+    if @board.update({title: params[:title]})
       redirect_to boards_path
     else
       render :edit
     end
   end
 
+  def destroy
+    Board.find(params[:id]).destroy
+    flash[:success] = "Board deleted"
+    redirect_to boards_path
+  end
+
+  def show
+    @show_id = params[:id]
+    @show_title = params[:title]
+    @find_id = Board.find(@show_id.to_i)
+  end
```

Comment:
tutaj to totalnie za duzo szukałaś... Po pierwsze skad sie wzal tutaj title ! :D powinnas tylko znalezc obiekt Board i z niego mozesz wyciagnac title.